molecule/default/hosts.yml

```diff
@@ -97,14 +105,30 @@ all:
             core-2:
             my-stateboard:
 
-        host2:
+        machine2:
           vars:
             ansible_host: vm2
             cartridge_package_path: ./packages/myapp-1.0.0-0.deb
 
           hosts:
             core-1:
+
+        machine3:
+          vars:
+            ansible_host: vm3
+            cartridge_package_path: ./packages/myapp-1.0.0-0.tar.gz
+
+            cartridge_multiversion: true
```

Comment:
Maybe to create one more test for `cartridge_multiversion: false`

Reply:
Yes, we can create one more machine for that.

Reply:
Let's do it!